android/java/org/chromium/chrome/browser/vpn/BraveVpnPrefUtils.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.chromium.chrome.browser.vpn;
+
+import android.content.SharedPreferences;
+
+import org.chromium.base.ContextUtils;
+import org.chromium.chrome.R;
+
+public class BraveVpnPrefUtils {
+    public static final String PREF_BRAVE_VPN_FEATURE = "brave_vpn_feature";
+    public static final String PREF_BRAVE_VPN_CALLOUT = "brave_vpn_callout";
+    public static final String PREF_BRAVE_VPN_CALLOUT_SETTINGS = "brave_vpn_callout_settings";
+    public static final String PREF_BRAVE_VPN_SUBSCRIPTION_PURCHASE =
+            "brave_vpn_subscription_purchase";
+    public static final String PREF_BRAVE_VPN_HOSTNAME = "brave_vpn_hostname";
+    public static final String PREF_BRAVE_VPN_PURCHASE_TOKEN = "brave_vpn_purchase_token";
+    public static final String PREF_BRAVE_VPN_PRODUCT_ID = "brave_vpn_product_id";
+    public static final String PREF_BRAVE_VPN_PURCHASE_EXPIRY = "brave_vpn_purchase_expiry";
+    public static final String PREF_BRAVE_VPN_SERVER_REGIONS = "brave_vpn_server_regions";
+
+    public static boolean isBraveVpnBooleanPref(String pref, boolean defaultValue) {
```

Comment:
maybe it is possible to consider to  make all the prefs named above private and for each preference make getter and setter, so instead of

```
            if (BraveVpnPrefUtils.isBraveVpnBooleanPref(
                        BraveVpnPrefUtils.PREF_BRAVE_VPN_CALLOUT, true)
                            && !BraveVpnPrefUtils.isBraveVpnBooleanPref(
                                    BraveVpnPrefUtils.PREF_BRAVE_VPN_SUBSCRIPTION_PURCHASE, false)
```
will be something like
```
            if (BraveVpnPrefUtils.isCallout() && !BraveVpnPrefUtils.isSubscriptionPurchase())
```
?
and we will not have to put the default value each time, so we will not be able to miss accidentally, and the expression will be more compact.

Reply:
@AlexeyBarabash i have implemented it that way to avoid many getter and seeter methods. and as you can see above we can use the same method for different prefs. please advise if i should change it.

Reply:
@deeppandya I think, that if you will have a separate getter/setter for the prefs, then the conditions in other files will become more easy to read. 

But this is an area of tastes and habits, so I don't insist on the change.

Reply:
@AlexeyBarabash i think i know what you mean. and i agree with you in that regards. i will update it. thank you for pointing out.

Reply:
thanks